app/services/coronavirus_pages/timeline_entry_builder.rb

```diff
@@ -0,0 +1,29 @@
+class CoronavirusPages::TimelineEntryBuilder
+  def create_timeline_entries
+    if timeline_entries_from_yaml.any?
+      TimelineEntry.transaction do
+        coronavirus_page.timeline_entries.delete_all
+
+        timeline_entries_from_yaml.each do |entry|
+          TimelineEntry.create!(
```

Comment:
Minor, this could be a little simpler with `coronavirus_page.timeline_entry.create!(` and you can save specifying the coronavirus_page.id.

Reply:
i did try this, having copied from https://github.com/alphagov/collections-publisher/commit/116f59fcdd61e8bbd3a4a301e8ed366fee03b7ab couldn't get it working for the life of me. Just get an error along the lines of 
`undefined method `timeline_entry' for #<CoronavirusPage:0x00005629a55ac1f0>`

Reply:
Oops I typo'd it's `coronavirus_page.timeline_entries` (plural not singular)

Reply:
nice one, sorted thanks :)